### ISSUE TITLE
multi GPU using nn.DataParallel error while after saving model during every epoch

### ISSUE THREAD

**Issue by alanli2000tw** [pranjaldatta/DenseDepth-Pytorch#12]
due to memory size, I'm trying to using 2gpu to train model with batch size 8. Seting model using pytorch nn.DataParallel func
facing an error " Caught RuntimeError in replica 0 on device 0 " while after saving the model during every epoch training, i'm quite confused and currrently still learning using Pytorch.

Have you got any possible solution to solve it? The model training under batch size 4 is not quite compatiable to orignal paper's evluation score, I want to check whether if it's possible to have the same accuracy as the orginial paper using an batch size 8.

Thank you.
